serde_json_path_core/src/spec/integer.rs

```diff
@@ -11,7 +11,7 @@ use std::{
 
 /// An integer for internet JSON ([RFC7493][ijson])
 ///
-/// The value must be within the range [-(2<sup>53</sup>)+1, (2<sup>53</sup>)-1]).
+/// The value must be within the range [-(2<sup>53</sup>)+1, (2<sup>53</sup>)-1].
```

Comment:
I assume the trailing `)` here was unintentional since it seems there is no opening `(`.

Reply:
Yep, that's a typo!